airsonic-main/src/main/java/org/airsonic/player/service/search/AnalyzerFactory.java

```diff
@@ -99,13 +103,13 @@ private void addTokenFilterForTokenToDomainValue(Builder builder) throws IOExcep
                     "pattern", "\\[\\]", "replacement", "\\[ \\]", "replace", "all");
     }
 
-    private Builder createDefaultAnalyzerBuilder() throws IOException {
+    private Builder createDefaultAnalyzerBuilder(boolean isArtist) throws IOException {
```

Comment:
It seems better to leave `createDefaultAnalyzerBuilder` alone and create a new `createArtistAnalyzerBuilder` method to be consistent with how you've handled the genere scenario.

Reply:
I revised that according to your advice.